### ISSUE TITLE
[Question] My Qu  ERROR: Failed building wheel for scikit-learn, how to fix it on windows?

### ISSUE THREAD

**Issue by kevinyongfy** [automl/auto-sklearn#1320]
# Short Question Description
A clear single sentence question we can try to help with?

With some extra context to follow it up. This way the question is clear for both you and us without it being lost in the paragraph. 
Some useful information to help us with your question:
* How did this question come about? 
* Would a small code snippet help?
* What have you already looked at?

Before you ask, please have a look at the
* [Documentation](https://automl.github.io/auto-sklearn/master/manual.html)
    * If it's related but not clear, please include it in your question with a link, we'll try to make it better!
* [Examples](https://automl.github.io/auto-sklearn/master/examples/index.html) 
    * Likewise, an example can answer many questions! However we can't cover all question with examples but if you think your question would benefit from an example, let us know!
* [Issues](https://github.com/automl/auto-sklearn/issues?q=label%3Aquestion+)
    * We try to label all questions with the label `Question`, maybe someone has already asked. If the question is about a feature, try searching more of the issues. If you find something related but doesn't directly answer your question, please link to it with #(issue number)!

# System Details (if relevant)
* Which version of `auto-sklearn` are you using?
* Are you running this on Linux / Mac / ... ?

**Comment by kevinyongfy** [comment]
ERROR: Failed building wheel for scikit-learn
Failed to build scikit-learn
ERROR: Could not build wheels for scikit-learn, which is required to install pyproject.toml-based projects

C:\WINDOWS\system32>pip install -r pyproject.toml
ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'pyproject.toml'

C:\WINDOWS\system32>python -m pip install -r pyproject.toml
ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'pyproject.toml'

**Comment by kevinyongfy** [comment]
How to fix it, what should I do?

**Comment by eddiebergman** [comment]
Hi @kevinyongfy,

This is more a question for scikit-learn and the error is quite clear, there is no `pyproject.toml` in your current directory `C:\WINDOWS\system32>`.

I'm closing this issue as it is not directly related to scikit learn.
